launcher/sdw_util/Util.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
This is great, thanks for doing this. There uptime, lockfile, but I think this definitely sufficient for merge in this iteration. If we were to move further (non-logging) logic here, we should have test coverage here

Reply:
I'll take a stab at refactoring the lockfile logic into the util library to further reduce duplication and increase testability.

Reply:
This is now done.